data/korath/korath weapons.txt

```diff
@@ -1107,7 +1107,7 @@ outfit "Firestorm Battery"
 		"corrosion damage" 5
 		"hit force" 4000
 		"missile strength" 115
-	description "Though not as refined as the newer Firelight version, this torpedo contains an augmented Korath Plasma Core reactor element completely stripped of its protective casing and safety interlocks. When activated, it generates a tremendous explosion of heat and force that pushes ships away and will even melt their hulls."
+	description "Though not as refined as the newer Firelight version, this Torpedo contains an augmented Korath Plasma Core reactor element completely stripped of its protective casing and safety interlocks. When activated, it generates a tremendous explosion of heat and force that pushes ships away and will even melt their hulls."
```

Comment:
```suggestion
	description "Though not as refined as the newer Firelight version, this torpedo contains an augmented Korath Plasma Core reactor element completely stripped of its protective casing and safety interlocks. When activated, it generates a tremendous explosion of heat and force that pushes ships away and will even melt their hulls."
```

This one is a bit confusing, because it looks like it's "Firelight Torpedo" broken up, but there is no such proper outfit, and since it is -not- referring to the human "Torpedo" outfit, it is just the term in general.